app/views/guide_typography.html

```diff
@@ -214,6 +215,23 @@ <h3 class="heading-medium" id="typography-hidden-text">Hidden text (progressive
   </div>
 
 
+  <h3 class="heading-medium" id="typography-class">Use styling automatically</h3>
+  <div class="text">
+    <p>You can use the <code class="code">govuk-prose</code> class around a body of prose. Then certain elements inside will be styled automatically without the need to add specific classes to each of them.</p>
+    <p>That will apply to most typographic elements: headings, lists, links (without govuk_template), code and strong. Tables, paragraphs, links (with govuk_template) and horizontal rules are already always styled without that class.</p>
```

Comment:
I'm not sure this is clear.

The new scope, `.govuk-prose` will apply styles to headings, lists, links and code and strong elements. 

We should include examples of each of these in the snippet. 

GOV.UK template sets [global link styles](https://github.com/alphagov/govuk_template/blob/master/source/assets/stylesheets/_basic.scss#L81), in addition to this GOV.UK elements sets global styles for [tables](https://github.com/alphagov/govuk_elements/blob/master/public/sass/elements/_tables.scss#L4), [paragraphs](https://github.com/alphagov/govuk_elements/blob/master/public/sass/elements/_elements-typography.scss#L138) and [horizontal rules](https://github.com/alphagov/govuk_elements/blob/master/public/sass/elements/_elements-typography.scss#L262).